src/FlipMove.js

```diff
@@ -119,21 +94,21 @@ class FlipMove extends Component {
     // longer, so that we can animate them away.
     this.setState({
       children: this.isAnimationDisabled(nextProps)
-        ? nextChildren
-        : this.calculateNextSetOfChildren(nextChildren)
+        ? nextChildren.map(element => ({ ...element, element }))
```

Comment:
We need to keep the reference to real `ReactElement`, because we pass it to `createElement`